### PR TITLE
Update Libraries-and-Plugins.html

### PR DESCRIPTION
**Pull request by atul-mourya**
**Description**

Added a newly released framework built on top of three.js. The framework is extensively documented, actively maintained, extendable, modular, and prioritizes high rendering quality.

Thanks to @repalash. I hope they don't mind its inclusion.

### CODE REVIEW

File: docs/manual/en/introduction/Libraries-and-Plugins.html [hunk 1]
```diff
@@ -106,6 +106,7 @@ <h3>Wrappers and Frameworks</h3>
 			<li>[link:https://aframe.io/ A-Frame]</li>
 			<li>[link:https://lume.io/ Lume] - HTML elements for 3D graphics built on Three.</li>
 			<li>[link:https://github.com/pmndrs/react-three-fiber react-three-fiber] - React components for 3D graphics built on Three.</li>
+			<li>[link:https://threepipe.org/ threepipe] - A versatile 3D viewer framework using three.js for rendering.</li>
```
Comment (Mugen87) [inline docs/manual/en/introduction/Libraries-and-Plugins.html:109]: Can you also add the new link to the translated pages?

Comment (atul-livspace) [reply docs/manual/en/introduction/Libraries-and-Plugins.html:109]: Sure

Comment (Mugen87) [reply docs/manual/en/introduction/Libraries-and-Plugins.html:109]: Never mind. I'll merge and clean up 👍 .

Comment (atul-mourya) [reply docs/manual/en/introduction/Libraries-and-Plugins.html:109]: Thanks :)

**Pull request merged**